core/jazz_assets/swagger/swagger.json

```diff
@@ -1145,29 +1185,139 @@
         }
       }
     },
-    "UpdateAssetsObj": {
-      "description": "Input object with set of fields that can be updated",
+    "AssetType": {
+      "type": "string",
+      "description": "type of the asset",
+      "enum": [
+        "lambda",
+        "apigateway",
+        "cloudfront",
+        "s3",
+        "dynamodb",
+        "cloudwatch_event",
+        "swagger_url",
+        "endpoint_url",
+        "apigee_proxy",
+        "dynamodb_stream",
+        "iam_role",
+        "sqs",
+        "kinesis_stream",
+        "sequence_diagram_url",
+        "certificate",
```

Comment:
let's remove the unsupported asset types - sequence_diagram_url, certificate & dns_record

Reply:
done

Reply:
Looks like these are not updated yet.

Reply:
Thanks for catching that. done.